axis.go

```diff
@@ -47,7 +47,7 @@ type Axis struct {
 		// For the vertical axis, one quarter turn
 		// counterclockwise will be added to the label
 		// text before drawing.
-		text.TextStyle
+		text.Style
```

Comment:
Can we keep this as `TextStyle`. This would me stopping it from being embedded, but do we actually use that?